projects/kit/directives/lazy-loading/lazy-loading.directive.ts

```diff
@@ -18,21 +33,35 @@ export class TuiLazyLoadingDirective {
         this.src$.next(src);
     }
 
+    animation = 'tuiSkeletonVibe ease-in-out 1s infinite alternate';
+
     @HostBinding('attr.src')
     src: string | null = null;
 
     private readonly supported = 'loading' in this.elementRef.nativeElement;
 
     constructor(
+        @Inject(TuiDestroyService)
+        private readonly destroy$: TuiDestroyService,
+        @Inject(ChangeDetectorRef) private readonly changeDetectorRef: ChangeDetectorRef,
         @Inject(TuiLazyLoadingService)
         private readonly src$: TuiLazyLoadingService,
         @Inject(ElementRef)
         private readonly elementRef: ElementRef<HTMLImageElement>,
     ) {
+        fromEvent(this.elementRef.nativeElement, 'load')
+            .pipe(takeUntil(this.destroy$))
```

Comment:
you can use `watch(changeDetectorRef)` inside pipe so you don't have to store it on `this`. Also since that would make `cancelSkeletonAnimation` just one line let's have it here directly.

Reply:
That is so neat! 😎

Reply:
That is so neat, done! Also I've removed unnecessary class scope from `destroy$` as well.